### ISSUE TITLE
Duplication of bulbs

### ISSUE THREAD

**Issue by kaiguy** [thkl/homebridge-philipshue#5]
Hi not sure if an issue but definitely a question.  Sorry if this is not the correct avenue to bring it up.

I noticed that when adding this module, all of my Philips hue bulbs (Homekit bulbs) are duplicated.  So if I have bulb1, bulb2, bulb3, bulb4, once I add this module, I now have 2 bulb1's, 2 bulb2's, and so on (I assume since both the Hue bridge and now Homebridge are exposing them separately).  Non-Homekit bulbs, however, only show once, since only homebridge is exposing them.

Edit: See recent comments here: https://github.com/nfarina/homebridge/issues/229

In short, new firmware on Philips Hue Bridge 2.0 re-adds 3rd party bulb compatibility, but closed the "bug" that allowed 3rd party bulbs to be controlled by Homekit/Siri.  This plugin to homebridge would be great for Hue Bridge 2.0 if the ability to exclude Hue bulbs was added.

**Comment by damianromanowski** [comment]
@kaiguy Check the pull request if you wish to set this up yourself, or wait until the author approves it.

**Comment by kaiguy** [comment]
@damianromanowski Thanks so much for working on this update! I'll give it a try when I get home this evening.  Really appreciate it!

**Comment by damianromanowski** [comment]
@kaiguy No prob, hopefully it works. I don't have any 3rd party bulbs on my bridge so there's no way for me to accurately test it - but in theory it should work.

**Comment by MR91** [comment]
It doesn't work for me. Are there any other things I need to change except the value in the config file?
I'm running Homebridge on a RPi.

**Comment by jjiburg** [comment]
See https://github.com/thkl/homebridge-philipshue/issues/7
